cmd/agent/main.go

```diff
@@ -160,19 +157,18 @@ E.g. kairos-agent install-bundle container:quay.io/kairos/kairos...
 		Aliases: []string{"i"},
 		Flags: []cli.Flag{
 			&cli.StringFlag{
-				Name:   "repository",
-				EnvVar: "REPOSITORY",
-				Value:  "docker://quay.io/kairos/packages",
+				Name:    "repository",
+				EnvVars: []string{"REPOSITORY"},
```

Comment:
nice, does this means you can set a flag linked to more than one env var? Pretty cool